lldb/include/lldb/Utility/ConstString.h

```diff
@@ -168,8 +168,8 @@ class ConstString {
   // Implicitly convert \class ConstString instances to \class StringRef.
   operator llvm::StringRef() const { return GetStringRef(); }
 
-  // Implicitly convert \class ConstString instances to \class std::string_view.
-  operator std::string_view() const {
+  // Explicitly convert \class ConstString instances to \class std::string_view.
+  explicit operator std::string_view() const {
```

Comment:
How about flipping this to make `llvm::StringRef` the explicit conversion and leave `std::string_view` the implicit one? This seems like it would be better as we tend towards the standard types, which would make the eventual removal of `StringRef` easier.

Reply:
I think that would be nice to do. FWIW, I tried making the StringRef conversion operator explicit a few months ago and found that it was a pretty large change. We use the implicitness of the conversion in a lot of places.

Reply:
Okay, I tried this to an extent and yeah it's a pretty large change :) I'll stick with the current way.

Reply:
:(